### DISCUSSION TITLE
Disable alternating users while queueing

### DISCUSSION THREAD

**Discussion by fezboy** [jagrosh/MusicBot#1110] [Questions]
Is it possible to turn off user alternation while queuing? It gets annoying with repeat on, because if one user has a lot of songs queued and another only has one or two the songs queued by the user with less will play half the time, even though they don't make up half of the songs.

For instance, if the queue looks like this, the song queued by user b will be every other song.
```
User A
User B
User A
User A
...
User A
```

**Comment by lorivaro** [comment]
This is FairQueue doing its job, and I believe this cannot be turned off.

**Comment by jagrosh** [comment]
There are currently no plans for disabling FairQueue.

That being said, I can understand the annoyance of it when in repeat mode. I think that a better solution than disabling FairQueue entirely would be a smarter approach to how songs are re-added to the queue when in repeat mode. For example, the "repeat" queue could be a separate queue that only began playing when the primary queue was empty.
